packages/components/src/custom-select-control-v2/stories/legacy.story.tsx

```diff
@@ -15,7 +15,7 @@ import CustomSelectControl from '../legacy-component';
 import * as V1Story from '../../custom-select-control/stories/index.story';
 
 const meta: Meta< typeof CustomSelectControl > = {
-	title: 'Components (Experimental)/CustomSelectControl v2/Legacy',
```

Comment:
Let's also do the same for the Default story. It was [unintentional](https://github.com/WordPress/gutenberg/pull/61111/files#r1579990587) that the Legacy story didn't have a WIP icon 🙈 

They're both WIP, but we can move them out of "Experimental" because they're marked with the badge.

Reply:
Cool, thanks for clarifying!